docs/user_guide/command_line_interface.rst

```diff
@@ -488,7 +489,7 @@ The available options:
     --format format, -f format
                             Data format to store retrieved embeddings
 
-The ``<trained_embedding_directory>`` is the directory created 
+The ``<trained_embedding_directory>`` is the directory ``<path.data_directory>/<general.experiment_name>`` created 
```

Comment:
Can we rename this directory to <model_directory>? In the code and in the docs.

Reply:
Do you mean by changing the current ``./data/`` to ``./model/`` or just the name used for this directory in ``marius_postprocess`` and ``marius_predict``?